subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -129,7 +129,7 @@ dependencies {
         api(libs.snakeyaml)             { version { strictly("1.28") }}
         api(libs.testng)                { version { strictly("6.3.1"); because("later versions break test cross-version test filtering") }}
         api(libs.tomlj)                 { version { strictly(tomljVersion) }}
-        api(libs.trove4j)               { version { strictly("1.0.20181211") }}
+        api(libs.trove4j)               { version { strictly("1.0.20200330") }}
```

Comment:
Interesting, does this need to be in sync with what the Kotlin plugin uses?

Reply:
It needs to be aligned with the embedded kotlin compiler.